media/materials/scripts/gazebo.material

```diff
@@ -240,6 +240,19 @@ material Gazebo/Black
   }
 }
 
+material Gazebo/BlackUnlit
+{
+  technique
+  {
+    lighting off
```

Comment:
this line is causing the `INTEGRATION_ogre_log` test to fail:

~~~
17:22:34: Compiler error: unknown error in gazebo.material(247): token "lighting" is not recognized
~~~

Reply:
thanks for catching this. Addressed in https://github.com/osrf/gazebo/pull/2840